### ISSUE TITLE
Exclusions

### ISSUE THREAD

**Issue by samhibberd** [someoneandsons/craft-ipwhitelist#1]
Would it be valuable to add exclusions, I would be great to allow certain urls / templates to be accessible to the public whilst activated??

**Comment by monachilada** [comment]
So certain routes would be configureable as not applying to the block?

**Comment by samhibberd** [comment]
Thats right yes, just allows you to open up certain areas which are still publicly accessible.

**Comment by monachilada** [comment]
I could see the value in it, yes. Currently I don't have time to dedicate to it, but I'd happily consider pull requests if you wanted to build that in yourself.

**Comment by samhibberd** [comment]
Sure thing, happy to. Such a useful plugin, the add id by url feature is a beauty!

**Comment by monachilada** [comment]
Yeah it's a real time saver. Thanks for the kind words, it means a lot. Looking forward to seeing what you come up with.
